infra/registry/instances.tf

```diff
@@ -0,0 +1,32 @@
+
+resource "openstack_compute_servergroup_v2" "servergroup" {
+  name     = "servergroup"
+  policies = ["anti-affinity"]
+}
+
+resource "openstack_compute_keypair_v2" "my-cloud-key" {
+  name       = "my-key"
+  public_key = var.key_pair
+}
```

Comment:
use keypair like this 
https://github.com/mekstack/mekstack/blob/7bc123e75740dc3247dec553838e5e91c45b5ede/infra/sneedaas/instances.tf#L12